### PR TITLE
Update CONTEXT.md

### LINKED ISSUE #845: Update Why DIANNA?
https://dianna.readthedocs.io/en/latest/CONTEXT.html

### PR DESCRIPTION
**Pull request by elboyran**
Why DIANNA in the documentation was obsolete.

### CODE REVIEW

**Comment by elboyran** [comment]
No idea, why the link checker thinks the TF link is dead- it isn't. I guess the dashboard test failing will be solved eventually by @laurasootes in #844 or related PR.

File: docs/CONTEXT.md [hunk 1]
```diff
@@ -2,54 +2,74 @@
...
+
  
 *	**Systematically chosen generic explainers**
 
 DIANNA includes few XAI approaches which have been evaluated using [systematically defined criteria](https://arxiv.org/ftp/arxiv/papers/1912/1912.05100.pdf). Using a relevant subset of these criteria, we have included complementary state-of-the-art XAI methods: [LIME](https://www.kdd.org/kdd2016/papers/files/rfp0573-ribeiroA.pdf), [RISE](http://bmvc2018.org/contents/papers/1064.pdf) and [KernelSHAP](https://proceedings.neurips.cc/paper/2017/file/8a20a8621978632d76c43dfd28b67767-Paper.pdf). These methods are model architecture agnostic.
 
+* **Simple unified interface**
+  
+DIANNA provided a simle unified interface to the chosen XAI methods (explainers). The library can be used both from the command-line and from its easy to use [dashboard](https://github.com/dianna-ai/dianna/tree/main?tab=readme-ov-file#dashboard).
```
Comment (cwmeijer) [inline docs/CONTEXT.md]: The dashboard is not ready to be used as an interface to DIANNA. It is merely a very cool demo tool at this point.
```suggestion
DIANNA provided a simple unified interface to the chosen XAI methods (explainers). The library can be used both from the command-line and we are working on an easy to use [dashboard](https://github.com/dianna-ai/dianna/tree/main?tab=readme-ov-file#dashboard), which can already be run with some of our examples for demo purposes.
```

File: docs/CONTEXT.md [hunk 2]
```diff
@@ -2,54 +2,74 @@
...
+
 *	**Standard DNN format**
 
 DIANNA is future-proof, the library is conforming with the [Open Neural Network eXchange (ONNX) standart](https://onnx.ai/). Using ONNX is not only very beneficial for interoperability, hence enabling reproducible-science, but ONNX-compatible runtimes and libraries are designed to maximize performance across hardware. 
 
 * **Multiple data modalities** 
 
-Most libraries support at most 2 modalities: images and text. While DIANNA has also started with these two, there are plans to include modalities, many scientists lack support for: time series, tabular data and embeddings. 
+Most libraries support at most 2 modalities: images and text. While DIANNA has also started with these two, now it also supports time series and tabular data with embeddings beging work in progress.
```
Comment (cwmeijer) [inline docs/CONTEXT.md]:
```suggestion
Most libraries support at most 2 modalities: images and text. While DIANNA has also started with these two, now it also supports time series and tabular data with embeddings being work in progress. 
```

File: docs/CONTEXT.md [hunk 3]
```diff
@@ -2,54 +2,74 @@
...
 
 * **Multiple data modalities** 
 
-Most libraries support at most 2 modalities: images and text. While DIANNA has also started with these two, there are plans to include modalities, many scientists lack support for: time series, tabular data and embeddings. 
+Most libraries support at most 2 modalities: images and text. While DIANNA has also started with these two, now it also supports time series and tabular data with embeddings beging work in progress. 
+
+*	**Simple datasets**
+
+To demonstrate the usefulness and enable studying the properties of the relevances on an intuitive level, DIANNA comes with new and exsisting [datasets](https://github.com/dianna-ai/dianna/tree/main?tab=readme-ov-file#datasets) and [models](https://github.com/dianna-ai/dianna/tree/main?tab=readme-ov-file#onnx-models) for both classification and regression tasks for image, text, time series and tabular data.
```
Comment (cwmeijer) [inline docs/CONTEXT.md]:
```suggestion
To demonstrate the usefulness and enable studying the properties of the relevances on an intuitive level, DIANNA comes with new and existing [datasets](https://github.com/dianna-ai/dianna/tree/main?tab=readme-ov-file#datasets) and [models](https://github.com/dianna-ai/dianna/tree/main?tab=readme-ov-file#onnx-models) for both classification and regression tasks for image, text, time series and tabular data.
```

**Review by cwmeijer — CHANGES_REQUESTED** [review]
Nice update indeed. I have some recommendation about the dashboard description, not being in a state for a user to work with their own model and/or data.

**Review by cwmeijer — APPROVED** [review]
👍

**Comment by elboyran** [comment]
It also depends on the outcome of #848.

**Pull request merged**